transport/transport.go

```diff
@@ -55,6 +62,13 @@ func (r *ResErrTuple) DiscardBody() {
 // returned value's response and error will be passed to client
 type MultipleResponsesHandler func(in <-chan ResErrTuple) ResErrTuple
 
+// Container mapping initialized Transports with http.RoundTripper by transport name
+type Container struct {
+	DefaultRoundTripper http.RoundTripper
```

Comment:
We can define a constant `defaultTransportName` and just use it on `RoundTrippers` map